src/middleware/indexer.js

```diff
@@ -162,18 +174,67 @@ const findLikelyNFTs = async (ctx) => {
             and action_kind = 'FUNCTION_CALL'
             and args->>'args_json' is not null
             and args->>'method_name' like 'nft_%'
+            and receipt_included_in_block_timestamp <= $2
```

Comment:
`actions_receipt_actions` has an index on `args->'args_json'->>'receiver_id'` but it doesn't reference `receipt_included_in_block_timestamp`. Can we see about getting the timestamp added to that index so that it's conducive to range queries? At the very least we should compare query plans between the current calls and this range query to rule it out as an issue - we don't want our well-intentioned change causing more DB contention.